README.md

```diff
@@ -220,6 +221,45 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
     msvcp140d_oss.dll
 ```
 
+# How To Run The Tests
+
+In order to run any of the tests you must be in a correctly configured environment for the architecture you are
+targeting, have python 3.8 or newer and have `clang-cl` on the PATH. If you are testing the x64 flavor of the STL then
+running from the "x64 Native Tools Command Prompt" is sufficient for setting up the environment.
+
+## Running All The Tests
+
+After configuring the project, running `ctest` from the root of the build output directory will run all the tests.
+CTest will only display the standard error output of tests which failed. In order to get more details from CTest's
+`lit` invocations, run the tests with `ctest -V`.
+
+## Running A Subset Of The Tests
+
+In order to run a subset of one of the testsuites you must invoke `lit` yourself and point it to the desired
+subdirectory of tests.
+
+The CMake configuration step produces a Python script, `${PROJECT_BINARY_DIR}\tests\llvm-lit\llvm-lit.py`.
+`llvm-lit.py` simply calls into the standard `lit` found in `llvm-project\llvm\utils\lit` with certain options
+pre-configured. Namely a mapping of `lit.cfg` files to `lit.site.cfg` files is provided, which informs lit which
+site-specific config file to load whenever it discovers a general config file.
+
+While one can directly use `lit` to run our tests by hand, we only support using the generated `llvm-lit`.
+
+### Examples
+
+```
+C:\Dev\STL\build>cmake -GNinja -DCMAKE_CXX_COMPILER=cl -DCMAKE_TOOLCHAIN_FILE=..\vcpkg\scripts\buildsystems\vcpkg.cmake ..
+
+:: This command will run all of the std testsuite.
+C:\Dev\STL\build>python tests\llvm-lit\llvm-lit.py tests\std
+
+:: However if you want to run a subset of std you need to point it to the right place in the sources.

```

Comment:
```suggestion
:: However, if you want to run a subset of std, you need to point it to the right place in the sources.
```
Or perhaps this should talk about running all configurations of one test in std?